app/workers/remove_assignment_worker.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+require "#{Rails.root}/lib/wiki_course_edits"
+
+class RemoveAssignmentWorker
+  include Sidekiq::Worker
+
+  def self.schedule_edits(course:, editing_user:, assignment:)
+    perform_async(course.id, editing_user.id, assignment.id)
+  end
+
+  def perform(course_id, editing_user_id, assignment_id)
+    course = Course.find(course_id)
+    editing_user = User.find(editing_user_id)
+    assignment = Assignment.find(assignment_id);
+    WikiCourseEdits.new(action: :remove_assignments, course: course, current_user: editing_user, assignment: assignment)
```

Comment:
Should be `:remove_assignment`

This breaks the UsersController spec. Run this to see if it works: `rspec ./spec/controllers/users_controller_spec.rb`